lib/engines/hcktest/drivers/NetKVM.json

```diff
@@ -5,7 +5,7 @@
   "install_method": "PNP",
```

Comment:
Don't use a long commit message title. The good practice to have the commit message title no longer than 72 characters. If this is needed you can add more information in the description.